### PR TITLE
1180 auto assign annots to encounter

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- If only one encounter in AGS, assign all annots to it on completion of detection

### CODE REVIEW

**Review by bluemellophone — APPROVED** [review]
LGTM, minus a likely typo

**Review by drewblount — APPROVED** [review]
LGTM

File: integration_tests/test_asset_group_sightings.py [hunk 1]
```diff
@@ -128,18 +128,7 @@ def test_asset_group_sightings(session, login, codex_url, test_root):
                         'customFields': sighting_custom_fields,
                         'decimalLatitude': -39.063228,
                         'decimalLongitude': 21.832598,
-                        'encounters': [
```
[the remark this answers is not in the document]
Comment (bluemellophone) [reply integration_tests/test_asset_group_sightings.py:131]: We can enforce determinism by throwing in a wait for the detection job, but I agree that the test itself is agnostic to the status of detection

**Review by bluemellophone — APPROVED** [review]
LGTM

**Pull request merged**